mlrun/common/schemas/alert.py

```diff
@@ -121,10 +132,15 @@ class AlertConfig(pydantic.BaseModel):
     trigger: AlertTrigger
     criteria: Optional[AlertCriteria]
     reset_policy: ResetPolicy = ResetPolicy.MANUAL
-    notifications: pydantic.conlist(Notification, min_items=1)
+    notifications: pydantic.conlist(AlertNotification, min_items=1)
     state: AlertActiveState = AlertActiveState.INACTIVE
     count: Optional[int] = 0
 
+    def get_notifications(self):
+        return [
+            alert_notification.notification for alert_notification in self.notifications
+        ]
```

Comment:
Why are you stripping away the cooldown period here? Is this because this method is used by someone expecting to get the raw notification? It would help to add a hint here (`-> list[Notification]`) and also call it differently, since now the object has a `notifications` property and `get_notifications` accessor which return different objects.

Reply:
Yes, some code needs the raw notifications